### DISCUSSION TITLE
Unity files seen as html in repo language stat bar

### DISCUSSION THREAD

**Discussion by c6tonsun** [github-linguist/linguist#5967] [Classification] [answered]
I made repo for unity project and now 87.4% is html and 12.6% C# in language stat bar. Only thing I can think of is that unity files (.meta .asset .unity ...) are seen as html.

What can I do to hide html from repo language? I have tried multiple ways with .gitattributes but no success yet. I'm running out of ideas and google search terms.

Best fix I can think of is to not detect unity file types as html :)

[Link to repo](https://github.com/c6tonsun/TunicTranslator)

**Comment by Nixinova** [comment, marked as answer]
The HTML comes from the big `.htm` file in the root of the repo.

If you want to hide it you can add this to the gitattributes:
`*.htm linguist-generated`

**Reply by c6tonsun** [reply]
Thank you I didnt notice that file as it doesnt show up elsewhere.
Its new file for me and I can ignore it as I use version control :)
